queries.sql

```diff
@@ -117,4 +117,9 @@ SELECT animals.name AS animal_name,animals.date_of_birth,animals.escape_attempts
 /*How many visits were with a vet that did not specialize in that animal's species?*/
 SELECT vets.name AS vet_name,animals.name AS animal_name,animals.species_id AS animal_species_id,specializations.species_id AS vets_specialization_id FROM visits  JOIN vets ON visits.vet_id = vets.id JOIN animals  ON visits.animal_id = animals.id JOIN specializations  ON specializations.vet_id = vets.id WHERE specializations.species_id != animals.species_id;
 /*What specialty should Maisy Smith consider getting? Look for the species she gets the most.*/
-SELECT vets.name AS vet_name,species.name AS animal_species,COUNT(species.name) AS number_of_animals FROM visits JOIN vets  ON visits.vet_id = vets.id JOIN animals ON visits.animal_id = animals.id JOIN species ON animals.species_id = species.id WHERE vets.name = 'Maisy Smith' GROUP BY species.name,vets.name ORDER BY number_of_animals DESC LIMIT 1;
\ No newline at end of file
+SELECT vets.name AS vet_name,species.name AS animal_species,COUNT(species.name) AS number_of_animals FROM visits JOIN vets  ON visits.vet_id = vets.id JOIN animals ON visits.animal_id = animals.id JOIN species ON animals.species_id = species.id WHERE vets.name = 'Maisy Smith' GROUP BY species.name,vets.name ORDER BY number_of_animals DESC LIMIT 1;
+
+/*Day one Performance*/
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
```

Comment:
- [ ] I can see that you tried your best to optimize this query:

```sql
EXPLAIN ANALYZE SELECT * FROM VISITS WHERE VET_ID = 2
```

Even if the execution times difference is very small in your case, I can clearly see that you had some fun with the `explain analyze` and I am sure that in the future you will know how to start the investigation for some slow DB queries